validation/src/test/scala/hmda/validation/rules/lar/macro/MacroSpec.scala

```diff
@@ -27,11 +27,32 @@ abstract class MacroSpec extends SummaryEditCheckSpec with BeforeAndAfterAll wit
     system.terminate()
   }
 
-  protected def newLarSource(lars: List[LoanApplicationRegister], numOfRelevantLars: Int, relevantDef: LoanApplicationRegister => LoanApplicationRegister, irrelevantDef: LoanApplicationRegister => LoanApplicationRegister) = {
-    val relevantLars = lars.map(lar => relevantDef(lar)).take(numOfRelevantLars)
-    val irrelevantLars = lars.map(lar => irrelevantDef(lar)).drop(numOfRelevantLars)
+  protected def newLarSource(
```

Comment:
... or just eliminating the need to further alter LARs after they're created. Could we just update the definition of `relevantLar` to fit the different test cases? (Not sure if that's a reasonable suggestion, since I haven't implemented it; I'm just brainstorming.)